packages/boot/test/bootstrapTests/test-vat-orchestration.ts

```diff
@@ -168,3 +181,65 @@ test('ICA connection can send msg with proto3', async t => {
     message: 'ABCI code: 5: error handling packet: see events for details',
   });
 });
+
+test('Query connection can be created', async t => {
+  const {
+    runUtils: { EV },
+  } = t.context;
+
+  const orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
+
+  const account = await EV(orchestration).createQueryConnection('connection-0');
+  t.log('Query Account', account);
+  t.truthy(account, 'createAccount returns an account');
+  t.truthy(
+    matches(account, M.remotable('QueryConnection')),
+    'QueryConnection is a remotable',
+  );
+});
+
+test('Query connection can send a query', async t => {
+  const {
+    runUtils: { EV },
+  } = t.context;
+
+  const orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
+
+  const queryConnection =
+    await EV(orchestration).createQueryConnection('connection-0');
+
+  const [result] = await EV(queryConnection).query([balanceQuery]);
+  t.truthy(result.code === 0);
+  t.truthy(typeof result.height === 'bigint');
+  t.truthy(
+    result.value.length === 0,
+    'XXX why do we get an empty Uint8Array here?',
```

Comment:
is this just XXX or do we need to resolve it before merge?

Reply:
I have removed this assertion for now